### PR TITLE
Added a create_csv function to the svgdigitizer and simplified the test filenames

### PR DESCRIPTION
**Pull request by DunklesArchipel**
This branch is based on the path-detection branch #25 from @linuxrider . I hope this works 🤞 

Added a `create_csv` function to the `svgdigitizer`. Related to #24 

Also added some files for some more generic tests (Related to #15). I also changed the tests, such that they use this version of `svgdigitizer`.

### CODE REVIEW

File: svgdigitizer/__main__.py [hunk 1]
```diff
@@ -12,8 +12,8 @@ def plot(svg):
 @click.command()
 @click.argument('basename')
 def digitize(basename):
-    from svgdigitizer.csvcreator import CreateCVdata
-    CreateCVdata(basename).create_csv()
+    from svgdigitizer.svgdigitizer import SvgData
+    CSvgData(basename).create_csv()
```
Comment (DunklesArchipel) [inline svgdigitizer/__main__.py:16]:
```suggestion
    SvgData(basename).create_csv()
```

**Comment by saraedum** [comment]
@DunklesArchipel could you try to resolve the merge conflicts? (Let me know if you need help with that.) Otherwise, it's a bit hard to review this.

**Comment by DunklesArchipel** [comment]
@saraedum I talked to Johannes before. He will finalize his work on the other branch (make the tests run properly). If this works, maybe you could have a brief look at that and merge it.

I will then fix all issues in this branch.

I also talked to Johannes about solving the merge conflicts. I guess this should be fine.

**Comment by saraedum** [comment]
Sure. Just ping me on zulip when you want me to look at something.

**Pull request merged**